packages/@apphosting/adapter-nextjs/src/bin/build.ts

```diff
@@ -1,38 +1,25 @@
 #! /usr/bin/env node
 import {
   loadConfig,
-  build,
   populateOutputBundleOptions,
   generateOutputDirectory,
-  DEFAULT_COMMAND,
   validateOutputDirectory,
 } from "../utils.js";
 import { join } from "path";
+import { getBuildOptions, runBuild } from "@apphosting/common";
 
 const root = process.cwd();
+const opts = getBuildOptions();
 
-let projectRoot = root;
-if (process.env.FIREBASE_APP_DIRECTORY) {
-  projectRoot = join(root, process.env.FIREBASE_APP_DIRECTORY);
-}
+// Set standalone mode
+process.env.NEXT_PRIVATE_STANDALONE = "true";
+// Opt-out sending telemetry to Vercel
+process.env.NEXT_TELEMETRY_DISABLED = "1";
+await runBuild();
```

Comment:
Do we want to support adapter authors who want to run a customized build command? Do we know that env vars are always sufficient to configure the build appropriately?

Reply:
Could you put together a small discovery doc of how other frameworks build commands work and whether this model would support them?

That would really help me understand the tradeoffs of this design

Reply:
+1 would love to see a doc that outlines this runBuild/ env var strategy so we can use it for our community framework strategy too.

That said I think it makes sense to publish this pr and continue the discussion elsewhere for now to unblock nx support.